src/pages/organize/[orgId]/people/duplicates/index.tsx

```diff
@@ -45,20 +59,39 @@ const DuplicatesPage: PageWithLayout = () => {
           </Typography>
         </Box>
       )}
-      {list.data && list.data.length > 0 && (
-        <Box p={1.5}>
+      {totalItems > 0 && (
+        <Box
+          ref={totalPages > 1 ? containerRef : undefined}
```

Comment:
Why this ternary here? Is there a problem with always assigning `containerRef`, even if there is no pagination?